skeleton/plugin/aurelia_project/tasks/build-plugin.ext

```diff
@@ -10,12 +10,13 @@ import {pluginMarkup} from './process-markup';
 import {pluginCSS} from './process-css';
 import {pluginJson} from './process-json';
 import {buildPluginJavaScript} from './transpile';
+import { CLIOptions } from 'aurelia-cli';
 
 function clean() {
-  return del('dist');
+  return del(['dist/**', '!dist']);
```

Comment:
It seems unnecessary. Is that removing of 'dist/' folder upsetting some of your tools/editors?

Reply:
yes, it creates some problems in rebuild. I will not delete dist if the --watch switch is enabled

Reply:
You probably can skip clean step in non-first build in watch mode.